contrib/utilities/indent

```diff
@@ -31,49 +46,49 @@ if test ! -f contrib/utilities/astyle.rc ; then
   exit 1
 fi
 
-if test -z "`which astyle`" ; then
+astyle="$(find_astyle)"
+astyle="${astyle:-`which astyle`}" # Fallback to `which astyle`
+
+if test -z "${astyle}"; then
   echo "*** No astyle program found."
   exit 1
 fi
 
-if test "`astyle --version 2>&1`" != "Artistic Style Version 2.04" ; then
+if [[ "$("${astyle}" --version 2>&1)" != "Artistic Style Version 2.04" ]]; then
   echo "*** Found a version of astyle different than the required version 2.04."
+  echo "  ${astyle}: $("${astyle}" --version)"
   exit 1
 fi
 
+echo "--- Found astyle 2.04 at ${astyle}"
+
 
 # collect all header and source files and process them in batches of 50 files
 # with up to 10 in parallel
 echo "--- Indenting all ASPECT header and source files"
 
-find tests include source benchmarks cookbooks unit_tests \( -name '*.cc' -o -name '*.h' -o -name '*.h.in' \) -print | xargs -n 50 -P 10 astyle --options=contrib/utilities/astyle.rc
+find tests include source benchmarks cookbooks unit_tests \( -name '*.cc' -o -name '*.h' -o -name '*.h.in' \) -print | xargs -n 50 -P 10 "${astyle}" --options=contrib/utilities/astyle.rc
 
 # remove execute permission on source files:
 find tests include source benchmarks cookbooks unit_tests \( -name '*.cc' -o -name '*.h' -o -name '*.prm' \) -print | xargs -n 50 -P 10 chmod -x
 
-# convert dos formatted files to unix file format by stripping out 
+# convert dos formatted files to unix file format by stripping out
 # carriage returns (15=0x0D):
-dos_to_unix()
-{
-    f=$1
-    tr -d '\015' <$f >$f.tmp
-    diff -q $f $f.tmp >/dev/null || mv $f.tmp $f
-    rm -f $f.tmp
-}
-export -f dos_to_unix
-find tests include source unit_tests cookbooks benchmarks \( -name '*.cc' -o -name '*.h' -o -name '*.prm' -o -name '*.h.in' \) -print | xargs -n 1 -P 10 -I {} bash -c 'dos_to_unix "$@"' _ {} 
+find tests include source unit_tests cookbooks benchmarks \( -name '*.cc' -o -name '*.h' -o -name '*.prm' -o -name '*.h.in' \) -print0 2>/dev/null \
+| while IFS=  read -r -d $'\0' src; do
```

Comment:
This change is unfortunate because it removes the `xargs -P 10` part and so treats file sequentially.

Reply:
Updated loop contents to execute all in parallel. I'm not sure if limiting the number of parallel processes to 10 is important here. It doesn't seem to use a significant amount of memory on my computer, but parallelizing the whole loop did speed the script up from 20 seconds (fully serial) to 3 seconds (fully parallel).